cypress/support/validations/screenshotValidation.js

```diff
@@ -0,0 +1,37 @@
+const CONSTANTS = require('../constants/constants');
+const UTILS = require('../cypress-support/src/utils');
+
+/**
+ * @module screenshotValidation
+ * @function screenshotValidation
+ * @description To validate the content of the response if undefined
+ * @param {Object} object - The object to validate
+ * @example
+ * cy.screenshotValidation({})
+ */
+Cypress.Commands.add('screenshotValidation', (object) => {
+  if (UTILS.getEnvVariable('enableScreenshots')) {
+    if (object && object.validations) {
+      const requestMap = {
+        method: 'fcs.screenshot',
+        params: {
+          addToReport: true,
```

Comment:
"addToReport" should be retrieved from the validation object itself.

Reply:
updated